lib/qtermwidget.h

```diff
@@ -41,6 +41,21 @@ class QTermWidget : public QWidget {
         ScrollBarRight=2
     };
 
+    enum KeyboardCursorShape {
```

Comment:
Do you suggest moving the enum declaration from `TerminalDisplay.h` to `qtermwidget.h`?
`enum ScrollBarPosition` is also duplicated.